siliconcompiler/tools/vpr/pack-place-route.py

```diff
@@ -0,0 +1,38 @@
+import os
```

Comment:
naming convention? I guess we don't have one, does `-` matter in python module names?

Reply:
Sort of - `import pack-place-route` is invalid, but it's okay to have in a filename.

This task name was already set, but I could `s/-/_/g`, or rename it to something like `apr`.

Reply:
rename might be better to avoid needing all these search and replaces in the tool

Reply:
Okay - but it might be best to merge #1211 before we modify the VPR flowgraph in this PR.

Reply:
@petergrossmann21 do you have any preference ?

Reply:
I vote for the task being 'apr'. (automated place and route ). It's pseudo standardized. The first time I heard about packing was with vpr.